src/components/NavBarButton.tsx

```diff
@@ -0,0 +1,32 @@
+import Link from "next/link";
+import { Button } from "./ui/button";
+
+interface NavBarButtonProps {
+  href: string;
+  text: string;
+  variant?: string;
```

Comment:
There’s no need to do this, but we can get Intellisense to autocomplete the types if we add it here instead of inside the component:
```suggestion
  variant?:
    | "default"
    | "link"
    | "destructive"
    | "outline"
    | "secondary"
    | "ghost";
```